src/groovy/au/org/emii/gogoduck/job/Job.groovy

```diff
@@ -37,6 +37,10 @@ class Job {
         new URL("$serverUrl/aggr/$uuid")
     }
 
+    URL getStatusUrl() {
+        new URL("$serverUrl/job/$uuid")
```

Comment:
The URL looks ok to me?  See https://github.com/aodn/go-go-duck/blob/master/grails-app/conf/UrlMappings.groovy#L6  Or can you see a specific problem with it @danfruehauf?